frontend/angular/style-guide.md

```diff
@@ -0,0 +1,972 @@
+      'href',
+      'https://wolox.com.ar'
+    );
+  }
+}
+```
+
+## Application structure
+ 
+Your application's folder structure should be similar to the following folder tree.
+
+```
+├── app
+│   ├── components
+│   │   ├── generic-component 
+│   │   │   ├── generic-component.component.html
+│   │   │   ├── generic-component.component.scss
+│   │   │   ├── generic-component.component.ts
+│   │   │   ├── generic-component.component.spec
+│   │   │   └── generic-component.module.ts
+│   │   ├── other-component
+│   │   │   ├── other-component.component.html
+│   │   │   ├── other-component.component.scss
+│   │   │   ├── other-component.component.ts
+│   │   │   ├── other-component.component.spec
+│   │   │   └── other-component.module.ts
+│   ├── helpers
+│   │   ├── classes
+│   │   ├── directives
+│   │   ├── enums
+│   │   ├── guards
+│   │   ├── pipes
+│   │   └── utilities
+│   ├── interfaces
+│   │   └── global.interface.ts
+│   ├── screens 
+│   │   └── Home
+│   │       ├── components 
+│   │       │   └── specific-component 
+│   │       │       ├── specific-component.component.html
+│   │       │       ├── specific-component.component.scss
+│   │       │       ├── specific-component.component.ts
+│   │       │       ├── specific-component.component.spec.ts
+│   │       │       └── specific-component.module.ts
+│   │       ├── interfaces 
+│   │       │   └── specific.interface.ts
+│   │       ├── home.component.html
+│   │       ├── home.component.scss
+│   │       ├── home.component.spec.ts
+│   │       ├── home.component.ts
+│   │       ├── home-routing.module.ts
+│   │       └── home.module.ts
+│   └── services
+│       ├── api.service.ts
+│       └── local-store.service.ts
+├── app-routing.module.ts
+├── app.component.html
+├── app.component.scss
+├── app.component.spec.ts
+├── app.component.ts
+├── assets
+├── config
+├── environments
+└── styles 
+    ├── global.sccs
+    └── base
+        ├── buttons.sccs
+        └── forms.sccs
+```
+
+**Considerations:** 
+  - Make locating code intuitive, simple and fast
+  - Create a module and routing for each screen (for lazy routing)
+  - Each general component must have a module
+  - Do not repeat elements like interfaces, enumerations or functions if they are used more than once, include them in the helpers folder
+
+## Modules
+
+A module can be easily loaded in different places in your app. Also, the modules can be isolated for [testing](https://angular.io/guide/testing#angular-testbed) as a code unit.
+
+The module concept is based on some rules:
+
+- Small features.
+- Not injecting general services or external services.
+
+A module can contain a Component, Directive or Pipe. The key is: _If you need to reuse any of them, you need a shared module._
+
+### Module and Routing by screen
+
+A screen can contain some specific sections and components, so try to create module and routing.module by screen. Also remember in Angular the main concept is module.
```

Comment:
```suggestion
A screen can contain specific sections and components, so try to create a module and routing.module by screen. Keep in mind that in Angular the main concept is module.
```
Maybe instead of `the main concept is module` it should be `the main concept is modularization ` I think module is a specific object and modularization is the concept